src/base/Assert.hh

```diff
@@ -60,22 +80,30 @@
 //! \endcond
 
 #if CELERITAS_DEBUG && defined(__CUDA_ARCH__)
-#    define REQUIRE(x) CELER_CUDA_ASSERT_(x)
-#    define CHECK(x) CELER_CUDA_ASSERT_(x)
-#    define ENSURE(x) CELER_CUDA_ASSERT_(x)
+#    define REQUIRE(COND) CELER_CUDA_ASSERT_(COND)
+#    define CHECK(COND) CELER_CUDA_ASSERT_(COND)
+#    define ENSURE(COND) CELER_CUDA_ASSERT_(COND)
```

Comment:
Is the a visual difference (beside looking up the code) between those 3 outputs?

Reply:
Huh, I must've left that off in the first iteration 😅

Shift allows these macros to be toggled on/off which IMHO is a bit of overkill. I can take the opportunity to bulk up the macros (e.g. changing the assertion text to be `precondition failed`, `assertion failed`, `postcondition failed`).